end-to-end-tests/cypress/e2e/createTransfer.cy.ts

```diff
@@ -18,6 +17,8 @@ import projectAssignmentPage from 'cypress/pages/projectAssignment'
 import rationalePage from 'cypress/pages/rationale'
 import trustInformationProjectDatesPage from 'cypress/pages/trustInformationProjectDates'
 
+
```

Comment:
Can we get rid of the unnecessary whitespacing please

(One for the future - we probably need to be making use of [the styling eslint plugin](https://eslint.style/packages/js#stylistic-eslint-plugin-js) across the board)

Reply:
done :)